xhr/resources/auth1/auth.py

```diff
@@ -1,10 +1,12 @@
 import imp
 import os
 
-here = os.path.split(os.path.abspath(__file__))[0]
+from wptserve.utils import isomorphic_decode
+
+here = os.path.split(os.path.abspath(isomorphic_decode(__file__)))[0]
 
 def main(request, response):
-    auth = imp.load_source("", os.path.join(here,
-                                            "..",
-                                            "authentication.py"))
+    auth = imp.load_source(u"", os.path.join(here,
```

Comment:
It looks like getting this import right is the source of most of the complexity, and that `imp.load_source` is only used in these tests and in service-workers/, apart from tools/. Still I don't know if there is a better way, searching I've come up empty-handed and think that adding a `__init__.py` and using `from .. import authentication` might be the only way.

@Hexcles @jgraham is there some way to clean this up to avoid all of the path juggling here?

Reply:
Ping @Hexcles @jgraham

Reply:
I've filed https://github.com/web-platform-tests/wpt/issues/24588 about this to unblock this PR.